net/server.go

```diff
@@ -409,3 +409,47 @@ func (s *server) updateReplicators(evt event.Replicator) {
 	}
 	s.peer.bus.Publish(event.NewMessage(event.ReplicatorCompletedName, nil))
 }
+
+func (s *server) AddPubSubTopic(topicName string, handler rpc.MessageHandler) error {
```

Comment:
todo: `addPubSubTopic` already exists. You can simply call it from this function if you need it to be public.